### PR TITLE
Modify Approvers list as Pavol now works for Traceable AI

### PR DESCRIPTION
**Pull request by kenfinnigan**

### CODE REVIEW

**Review by arminru — APPROVED** [review]

**Comment by carlosalberto** [comment]
Out of curiosity, @pavolloffay are you still planning to stay involved, now that you changed employer?

**Review by carlosalberto — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ See [CONTRIBUTING.md](CONTRIBUTING.md)
 Approvers ([@open-telemetry/java-approvers](https://github.com/orgs/open-telemetry/teams/java-approvers)):
 
 - [Armin Ruech](https://github.com/arminru), Dynatrace
-- [Pavol Loffay](https://github.com/pavolloffay), RedHat
+- [Pavol Loffay](https://github.com/pavolloffay), Traceable AI
```
Comment (jkwatson) [inline README.md:148]: @pavolloffay isn't it just "Traceable" ?

Comment (kenfinnigan) [reply README.md:148]: I went off his LinkedIn, if that's wrong I can adjust

Comment (jkwatson) [reply README.md:148]: if that's what it says on LinkedIn, then it's probably correct. :) I just looked at their website, and it just says Traceable everywhere, which is what I based my comment on. :)

**Review by anuraaga — APPROVED** [review]

**Comment by anuraaga** [comment]
Will let pavol approve before merging :)

**Review by thisthat — APPROVED** [review]

**Review by pavolloffay — APPROVED** [review]
LGTM

I would rather put there `Traceable.ai`

**Comment by pavolloffay** [comment]
> Out of curiosity, @pavolloffay are you still planning to stay involved, now that you changed employer?

Yes,  I should get more involved with the Java ecosystem.

File: README.md (hunk 1 above)
Comment (arminru) [reply README.md]: https://github.com/open-telemetry/opentelemetry-java/pull/1862#pullrequestreview-515678775 --> 4fee432

**Pull request merged**